src/bin/node.rs

```diff
@@ -58,31 +59,29 @@ async fn main() -> Result<()> {
 
     info!("server mode");
 
-    let mut mp = Mempool::new(args.id, config.mempool_peers.clone());
-    let (sender, receiver) = tokio::sync::mpsc::unbounded_channel::<Transaction>();
+    let mut mp = Mempool::new();
+    let (mempool_tx, mempool_rx) = tokio::sync::mpsc::unbounded_channel::<MempoolMessage>();
     tokio::spawn(async move {
-        _ = mp.start(receiver).await;
+        _ = mp.start(mempool_rx).await;
     });
 
-    let (consensus_tx, rx) = mpsc::channel::<ConsensusCommand>(100);
+    let (consensus_tx, consensus_rx) = mpsc::channel::<ConsensusCommand>(100);
 
-    start_consensus(consensus_tx.clone(), rx, sender, config.clone());
+    start_consensus(
+        consensus_tx.clone(),
+        consensus_rx,
+        mempool_tx.clone(),
+        config.clone(),
+    );
 
-    if args.no_rest_server {
```

Comment:
why remove this logic ? If not needed remove this to Args struct maybe ?

Reply:
it was used when we had one config file with all peers inside